spec/lib/search_spec.rb

```diff
@@ -7,6 +7,7 @@
   before do
     SearchIndexer.enable
     Jobs.run_immediately!
+    SiteSetting.enable_whispers = true
```

Comment:
Instead of enabling this for the entire spec, I think it'll be better to only enable it in the specs which requires it.